src/logger.service.ts

```diff
@@ -27,77 +25,101 @@ const Levels = [
 
 @Injectable()
 export class NGXLogger {
-
   private _serverLogLevel: NgxLoggerLevel;
   private _clientLogLevel: NgxLoggerLevel;
   private _isIE: boolean;
 
-  constructor(private http: HttpClient, @Inject(PLATFORM_ID) private platformId, @Optional() private options: LoggerConfig) {
-    if (!this.options) {
-      this.options = {
-        level: NgxLoggerLevel.OFF,
-        serverLogLevel: NgxLoggerLevel.OFF
-      };
-    }
+  public constructor(
```

Comment:
are there actually any changes here? or just reformatting? please revert all reformatting changes, so that the changes are easily identifiable.